### PR TITLE
Make IsValidLabelValue return error strings

### PR DESCRIPTION
**Pull request by thockin**
Part of the larger validation PR, broken out for easier review and merge.  Builds on previous PRs in the series.

### CODE REVIEW

File: pkg/kubelet/network/plugins.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func InitNetworkPlugin(plugins []NetworkPlugin, networkPluginName string, host H
 	allErrs := []error{}
 	for _, plugin := range plugins {
 		name := plugin.Name()
-		if !validation.IsQualifiedName(name) {
+		if len(validation.IsQualifiedName(name)) != 0 {
 			allErrs = append(allErrs, fmt.Errorf("network plugin has invalid name: %#v", plugin))
```
Comment (lavalamp) [inline pkg/kubelet/network/plugins.go]: Shouldn't you append w/ the actual errors?

File: pkg/kubectl/cmd/label.go [hunk 2]
```diff
@@ -124,9 +124,12 @@ func parseLabels(spec []string) (map[string]string, []string, error) {
 	for _, labelSpec := range spec {
 		if strings.Index(labelSpec, "=") != -1 {
 			parts := strings.Split(labelSpec, "=")
-			if len(parts) != 2 || len(parts[1]) == 0 || !validation.IsValidLabelValue(parts[1]) {
+			if len(parts) != 2 || len(parts[1]) == 0 {
 				return nil, nil, fmt.Errorf("invalid label spec: %v", labelSpec)
 			}
+			if len(validation.IsValidLabelValue(parts[1])) != 0 {
```
Comment (lavalamp) [inline pkg/kubectl/cmd/label.go]: the pattern should be

```go
if errs := validation.IsValidFrobber(frobber); len(errs) != 0 {
  // do something useful with errs
}
```

**Comment by lavalamp** [comment]
Concept seems fine, but there's a few places (not all marked) where the callers are throwing away the new errors instead of passing them up the chain.

File: pkg/kubelet/network/plugins.go (hunk 1 above)
Comment (thockin) [inline pkg/kubelet/network/plugins.go]: some were from previous PR, but sure, done.

**Comment by thockin** [comment]
rebased and pushed

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**